addons/udes_stock_cron/data/ir_cron.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record id="reserve_stock_action" model="ir.cron">
+        <field name="name">Reserve stock</field>
+        <field name="active" eval="True" />
```

Comment:
Does the DO want this disabled? It is not clear to me

Reply:
DO needs clarifying for sure. I would expect the cron job to be on by default, but disabled for all picking types by default. We should clarify this with product.